packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -63,12 +63,49 @@ class Autocomplete<T extends Object> extends StatelessWidget {
     required this.optionsBuilder,
     this.displayStringForOption = RawAutocomplete.defaultStringForOption,
     this.fieldViewBuilder = _defaultFieldViewBuilder,
+    this.focusNode,
     this.onSelected,
+    this.textEditingController,
     this.optionsMaxHeight = 200.0,
     this.optionsViewBuilder,
     this.optionsViewOpenDirection = OptionsViewOpenDirection.down,
     this.initialValue,
-  });
+  }) : assert(
+         fieldViewBuilder != null
```

Comment:
nit: `fieldViewBuilder` can never be null.

Reply:
I directly copied the assert logic from `RawAutocomplete` so that is where the `fieldViewBuilder` null check came from.  I will remove that.

Reply:
Yes the motivation for this change was so that the Material Autocomplete widget could be used but also retain the functionality of the underlying RawAutocomplete widget.

I was considering making a test using inspiration from the RawAutocomplete test you mentioned, I will take a look...
At the very least I plan to verify that the passed in focusNode and textEditingController are used within the RawAutocomplete.

Reply:
I think it's enough to just test that the parameters are passed through to RawAutocomplete correctly :+1: